scripts/qa_crowbarsetup.sh

```diff
@@ -659,8 +659,8 @@ EOF
     fi
 
     if [ -n "${localreposdir_target}" ]; then
-        for x in `seq 1 10` ; do
-            zypper rr 1
+        while zypper rr 1 2>&1 | grep -v "^Repository '1' not found"; do
+            :
```

Comment:
I already thought about doing that but decided against it because it violates the Law of Demeter.  It's cleaner to treat libzypp as a black box and go through an intended user interface.